packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -324,10 +344,19 @@ export default class ReconnectionManager {
       throw (new NeedsRetryError(syncError));
     }
 
+    // TODO: try to improve this logic as the reconnection manager saves the instance of deleted meeting object
+    // So that on rejoin it known what parametrs it was using
+    if (!this.meeting || !this.webex.meetings.getMeetingByType(_ID_, this.meeting.id)) {
+      LoggerProxy.logger.info('ReconnectionManager:index#executeReconnection --> Meeting got deleted due to inactivity or ended remotely ');
+
+      throw new Error('Unable to rejoin a meeting already ended or inactive .');
+    }
+
     LoggerProxy.logger.info(`ReconnectionManager:index#executeReconnection --> Current state of meeting is ${this.meeting.state}`);
 
     // If the meeting state was left, no longer reconnect media
-    if (this.meeting.state === _LEFT_) {
+    // or the meeting was deleted in process of re connecting
+    if (this.meeting.state === _LEFT_ || this.meeting.state === _INACTIVE_) {
```

Comment:
I'm a bit confused by this. I thought the whole point of the new autoRejoin field in config was to rejoin when we are in "inactive" state, but here we don't allow rejoining in "inactive" state.

Reply:
I think this line wont be executed at all . Removing it 

Before line 347 was added reconnect was in progress but the meeting would have already ended before the reconnect promise would finish

Reply:
The rejoin toggle makes sure that you need to trigger the rejoin compared to SDK doing a rejoin for you